backend/alembic/versions/6d107741a92e_add_glossary.py

```diff
@@ -0,0 +1,51 @@
+"""add glossary
+
+Revision ID: 6d107741a92e
+Revises: b3e764c93fac
+Create Date: 2024-07-17 01:01:34.146777
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pylint: disable=E1101
+
+# revision identifiers, used by Alembic.
+revision: str = '6d107741a92e'
+down_revision: Union[str, None] = 'b3e764c93fac'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('glossary_document',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('created_by', sa.Integer(), nullable=False),
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.ForeignKeyConstraint(['created_by'], ['user.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('glossary_record',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.Column('document_id', sa.Integer(), nullable=False),
+    sa.Column('comment', sa.String(), nullable=False),
+    sa.Column('en_term', sa.String(), nullable=False),
+    sa.Column('ru_term', sa.String(), nullable=False),
```

Comment:
It should be src and dst terms to be consistent with other parts of DB

Reply:
Done